protocols/mdns/src/behaviour/iface.rs

```diff
@@ -200,51 +200,58 @@ where
         cx: &mut Context,
         params: &impl PollParameters,
     ) -> Option<(PeerId, Multiaddr, Instant)> {
```

Comment:
Not directly related. Why do we return an `Option` and not a `Poll`?

Reply:
I don't know why.

Reply:
Fixed it!